app/__init__.py

```diff
@@ -46,6 +46,19 @@ def add_cors_header(response):
     return response
 
 
+# Set Cache Headers
+@app.after_request
+def add_cache_header(response):
+    # overwrite with these 5xx cache settings
```

Comment:
You could add a check to only add a Cache Control header for GET, HEAD and OPTIONS method.

Reply:
@ltshb yes, its done!